modules/home-manager/zed-editor.nix

```diff
@@ -2,30 +2,47 @@
 { config, lib, ... }:
 
 let
+  inherit (config.catppuccin) sources;
+
   cfg = config.catppuccin.zed;
 in
 
 {
-  options.catppuccin.zed = catppuccinLib.mkCatppuccinOption { name = "zed"; } // {
-    italics = lib.mkEnableOption "the italicized version of theme" // {
-      default = true;
+  options.catppuccin.zed =
+    catppuccinLib.mkCatppuccinOption {
+      name = "zed";
+      accentSupport = true;
+    }
+    // {
+      italics = lib.mkEnableOption "the italicized version of theme" // {
+        default = true;
+      };
     };
-  };
 
   config = lib.mkIf cfg.enable {
     programs.zed-editor = {
-      extensions = [ "catppuccin" ];
-
       userSettings.theme = {
         light =
           "Catppuccin "
           + catppuccinLib.mkUpper cfg.flavor
```

Comment:
```suggestion
          "Catppuccin Latte"
```

Reply:
`programs.zed-editor.userSettings.theme.mode = dark` can be used to override to dark

Reply:
actually never mind: see #420